components/request_otr/browser/BUILD.gn

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/build/config.gni")
+import("//brave/components/request_otr/common/buildflags/buildflags.gni")
+
+source_set("browser") {
+  sources = []
+  deps = []
+
+  if (enable_request_otr) {
```

Comment:
This seems inconsistent with the way the buildflags are used in other places in the code. Normally we either encapsulate the buildflags within the component and return nullptr, etc... or everything gets wrapped in the buildflag. In this case everything is wrapped in the buildflag except for these targets so to keep things consistent I would wrap the deps in the buildflag and `assert(enable_request_otr)` in this file

Reply:
I may or may not have fixed this properly.